### ISSUE TITLE
Need to be clear on what the 30d free trial is when hitting quota

### ISSUE THREAD

**Issue by chrisdias** [microsoft/vscode-copilot-release#3658]
See https://x.com/nullnone_nil/status/1873666951823921319

We don't say what the 30d free trial is of/for.

![Image](https://github.com/user-attachments/assets/5ddbcd5c-66f8-4fb3-8319-7f8272bd051f)

**Comment by bpasero** [comment]
Neither does GH.com 🤔 . But isn't this pretty clear?

![Image](https://github.com/user-attachments/assets/feb4613f-cdc1-4ebe-b6a5-c220ae03f307)

Any suggestions are welcome about the wording.

//cc @isidorn @bamurtaugh

**Comment by chrisdias** [comment]
ok, so the first 30d of Pro are free, got it.

Suggestion: "Upgrade to Pro for unlimited Completions and Chat (your first 30 days are free)."

**Comment by bpasero** [comment]
@chrisdias to clarify, you suggest to use that sentence and thus replace this entire block?

<img width="612" alt="Image" src="https://github.com/user-attachments/assets/b95cac4f-c4be-43da-9bf1-b6413cb5b23b" />

**Comment by bamurtaugh** [comment]
> you suggest to use that sentence and thus replace this entire block?

This suggestion makes sense to me too!

**Comment by chrisdias** [comment]
Yes, a sentence.  Is the part I just added about "additional models" actually true?

`Upgrade to Pro (your first 30 days are free) for unlimited Completions, Chat, and access to additional models.`

**Comment by bpasero** [comment]
That is actually a good point that we did not make clear before: you get access to more models, including O1!

**Comment by bpasero** [comment]
@chrisdias @bamurtaugh Somehow I like the bullet list of things I get when upgrading, how about this?

![Image](https://github.com/user-attachments/assets/83242778-b5d8-4aa5-8b09-ad62701e12ec)

**Comment by chrisdias** [comment]
Love it. I would say "Access to additional models" but not a blocker.
